### ISSUE TITLE
[feature request] support the property `current-window-scale`

### ISSUE THREAD

**Issue by hooke007** [stax76/mpv.net#310]
**Is your feature request related to a problem? Please describe.**
The previous edition disable the supporting for `window-scale` for potential bugs.
mpv fix this issue by this commit. https://github.com/mpv-player/mpv/commit/a0441ddb5e1a494275dcb8235268cd113babc491
https://mpv.io/manual/master/#command-interface-current-window-scale

**Describe the solution you'd like**
To support the two properties again.

**Describe alternatives you've considered**
...

**Comment by stax76** [comment]
That is probably not easy to understand and also implement, simply because most window functionality has high complexity in general.

I would only do something if somebody can properly describe an important use case for mpv.net.

In mpv on Windows 10 `window-scale` does not work correctly btw., setting `window-scale=1` after startup is ignored which is a bug I'm too lazy to report, because of this bug mpv.net uses an own command, so it works correctly in mpv.net. Maybe it was fixed meanwhile, probably not, my mpv version isn't always the newest.

**Comment by hooke007** [comment]
>setting window-scale=1 after startup is ignored which is a bug I'm too lazy to report

No, it is clear in the manual (not a bug). Because the default value of `window-scale` is `1` , so set it to 1 after startup and nothing will happen. Only if you set the different value such as `window-scale=1.5` in the conf file , setting `window-scale=1` after startup would work as expectation.

https://mpv.io/manual/master/#command-interface-window-scale
`Note that setting a value identical to its previous value will not resize the window. That's because this property mirrors the window-scale option, and setting an option to its previous value is ignored.`

**Comment by hooke007** [comment]
>I would only do something if somebody can properly describe an important use case for mpv.net.

In mpv i use `Alt+-   set current-window-scale -0.5` and `Alt+=   add current-window-scale 0.5` to adjust the window size step by step. If is there similar function in mpvnet?

**Comment by stax76** [comment]
I understand it better now, thanks. This functionality was missing before in mpv and because of that, mpv.net uses an own method which works well. It's probably very hard to implement, mpv.net users can use:

```
Alt++   script-message mpv.net scale-window 1.2     #menu: View > Zoom > Enlarge
Alt+-   script-message mpv.net scale-window 0.8     #menu: View > Zoom > Shrink
```

**Comment by hooke007** [comment]
>Alt++   script-message mpv.net scale-window 1.2     #menu: View > Zoom > Enlarge
I knew it, but it could only do 1.2x1.2x1.2......step by step. I use plus (such as 1+0.2+0.2+0.2+.......) more than multiplier
